pilot/pkg/model/validation.go

```diff
@@ -268,7 +268,19 @@ func checkDNS1123Preconditions(name string) error {
 
 func validateDNS1123Labels(domain string) error {
 	parts := strings.Split(domain, ".")
-	topLevelDomain := parts[len(parts)-1]
+	lastPart := parts[len(parts)-1]
+	lastPartArray := strings.SplitN(lastPart, ":", 2)
+	
```

Comment:
@morvencao there are some extra spaces here in this line, should resolve the lint failure you are seeing

Reply:
@GregHanson Thanks for reminding me. It works.